### PR TITLE
feature/COR-1481-remove-safety-regions-page

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard21**
## Summary

* Removed Safety Regions page
* Removed Sanity key
* Removed VrComboBox
* Removed reference of VrComboBox in vr-layout.tsx

#### Before
<details>
<summary>Toggle before screenshots</summary>

Scherm­afbeelding 2023-03-23 om 12 57 15

Scherm­afbeelding 2023-03-23 om 12 57 29

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Scherm­afbeelding 2023-03-23 om 12 57 05

Scherm­afbeelding 2023-03-23 om 12 56 26

</details>

### CODE REVIEW

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Looks awesome!

File: packages/app/src/pages/veiligheidsregio/index.tsx [hunk 1]
```diff
@@ -1,98 +0,0 @@
...
-    <Layout {...metadata} lastGenerated={lastGenerated}>
-      <VrLayout isLandingPage>
-        {!breakpoints.md && (
-          <Box bg="white">
-            <VrComboBox selectedVrCode={code} />
-          </Box>
-        )}
-
-        <Box as="article" padding={space[4]} spacing={3}>
-          {commonTexts.regionaal_index.belangrijk_bericht && <WarningTile message={commonTexts.regionaal_index.belangrijk_bericht} variant="emphasis" />}
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/pages/veiligheidsregio/index.tsx:59]: I think this common text can also be removed from Sanity

Comment (VWSCoronaDashboard25) [reply packages/app/src/pages/veiligheidsregio/index.tsx:59]: I think it should be removed

Comment (VWSCoronaDashboard21) [reply packages/app/src/pages/veiligheidsregio/index.tsx:59]: You are right, I have removed this Sanity key. Therefor I have resolved this comment.

**Review by VWSCoronaDashboard18 — DISMISSED** [review]
Nice job!

**Review by VWSCoronaDashboard25 — APPROVED** [review]

**Pull request merged**